resallocserver/maint.py

```diff
@@ -59,6 +60,19 @@ def resource_delete(self, resources=None):
                 resources = QResources(session=session)
                 resources.kill(res_id)
 
+    def resource_logs(self, resources=None):
+        hooks_dir = os.path.join(app.config["logdir"], "hooks")
+        paths = []
+        for resource in resources:
+            # We can't wildcard everything because then `tail' wouldn't
+            # discover newly created log files
+            suffixes = ["_alloc", "_watch", "_terminate"]
+            path = os.path.join(hooks_dir, resource.zfill(6))
+            paths.extend([path + suffix for suffix in suffixes])
+
+        cmd = ["tail", "-F"] + paths
```

Comment:
I usually use `-n+0`.  Not sure if that's what you want, perhaps it may be optional?

Reply:
I think it is not that important to me, I updated the code and added a non-optional `-n+0`. We can make it optional once it proves itself to be useful.